tools/builders/launcher-builder/1.10.1/cloudbuild.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+- name: 'gcr.io/cloud-builders/git'
```

Comment:
I don't think we need the clone steps here. It's just a Dockerfile that is being submitted. Let's have the makefile use `gcloud container builds submit --config cloudbuild.yaml .`.

Reply:
container can't see the Dockerfile without cloning the repo.

Reply:
Why not? it should get uploaded when you submit.

Reply:
actually looks like I may be able to pass "." to get it to work.